VTEX - Logistics API.json

```diff
@@ -7865,7 +7865,7 @@
                             "application/json": {
                                 "schema": {
                                     "type": "object",
-                                    "description": "Paged pickup points list.",
+                                    "description": "Paged pickup points list. The response is limited to 100 pages, but you can start the count from any number using the `page` query param.",
```

Comment:
```suggestion
                                    "description": "Paged pickup points list.",
```

Reply:
Regardless of the number of pages, only one page is returned in the response, so I don't think this information should be here.

I recommended adding the limit specifically to the `pages` parameter in the response, to clarify that even though there might be other results, the max value of this field in the response will be 100. That's line **7895**.

Reply:
Reopened this comment because despite the 👍 the changes were not applied.